packages/mdc-ripple/README.md

```diff
@@ -182,6 +182,23 @@ ripple.
 MDCRipple.attachTo(document.querySelector('.surface'));
 ```
 
+### Ripple API
+
+The component allows for programmatic activation / deactivation of the ripple, for complex interaction between deeply
```

Comment:
could we say "for interdependent interaction between components" instead of "for complex interaction between deeply tied components"?